src/txmempool.h

```diff
@@ -63,9 +63,10 @@ class CTxMemPoolEntry
     size_t nModSize; //! ... and modified size for priority
     size_t nUsageSize; //! ... and total memory usage
     int64_t nTime; //! Local time when entering the mempool
-    double dPriority; //! Priority when entering the mempool
```

Comment:
This was originally the first part of the changes necessary for #6357 (now #7149) 
It was necessary to distinguish between the two stored priorities and heights.  The original "entry" priority and height and the latest calculated "cached" priority and the height that was calculated at.

It seemed logical to me to leave those changes in given that we didn't know whether the second half would be merged or not and given that they are more accurate names anyway.